sqlitecluster/SQLitePool.cpp

```diff
@@ -87,11 +87,18 @@ void SQLitePool::returnToPool(size_t index) {
     _wait.notify_one();
 }
 
-SQLiteScopedHandle::SQLiteScopedHandle(SQLitePool& pool, size_t index) : _pool(pool), _index(index)
+SQLiteScopedHandle::SQLiteScopedHandle(SQLitePool& pool, size_t index) : _pool(pool), _index(index), _released(true)
```

Comment:
should this start with `false` instead? Or do we need somewhere to actually change that to false in the code?

Reply:
It definitely should.